Remotion/Web/Core/UI/Controls/ValidationError.cs

```diff
@@ -204,12 +204,21 @@ public HtmlGenericControl ToSpan (string? cssClass = null)
     /// <returns> A <see cref="HtmlGenericControl"/>. </returns>
     private HtmlGenericControl ToGenericControl (string? cssClass, string tag)
     {
-      HtmlGenericControl genericControl = new HtmlGenericControl(tag);
-      genericControl.EnableViewState = false;
+      HtmlGenericControl genericControl;
+      if (_validationMessage.IsEmpty)
+      {
+        genericControl = new LazyValidationControl(tag, _validator);
+      }
+      else
+      {
+        genericControl = new HtmlGenericControl(tag);
+        genericControl.InnerHtml = ValidationMessage.ToString(WebStringEncoding.HtmlWithTransformedLineBreaks);
+      }
```

Comment:
I don't think we need to leave the original logic in. Yes we delay accessing `ErorrMessage`, but its value should not change anyway. Since we create a control instance anyway, we simplify and use just our control.

Reply:
Talked about this with mike, he thinks we should leave the splitting of behaviour here.